### DISCUSSION TITLE
How to resolve this problem?

### DISCUSSION THREAD

**Discussion by jogsunzhao** [filecoin-project/lotus#4620] [Storage Provider]
sealing sectors no finished, how to resolve?

**Describe the problem**

pushing message to mpool: GasEstimateMessageGas error: estimating gas used: message execution failed: exit 16, reason: seal challenge epoch 180985 too old, must be after 181080 (RetCode=16)

**Sectors status**

7529.	2020-10-27 09:42:55 +0800 CST:	[event;sealing.SectorRetrySealPreCommit1]	{"User":{}}
7530.	2020-10-28 08:04:20 +0800 CST:	[event;sealing.SectorPreCommit1]	{"User":{"PreCommit1Out":"eyJyZWdpc3RlcmVkX3Byb29mIjoiU3RhY2tlZERyZzMyR2lCVjEiLCJsYWJlbHMiOnsiU3RhY2tlZERyZzMyR2lCVjEiOnsibGFiZWxzIjpbeyJwYXRoIjoiL2FwcHMvc3RvcmFnZS9sb3R1c3dvcmsvbG90dXN3b3JrZXIvY2FjaGUvcy10MDE0ODAzLTY1IiwiaWQiOiJsYXllci0xIiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvYXBwcy9zdG9yYWdlL2xvdHVzd29yay9sb3R1c3dvcmtlci9jYWNoZS9zLXQwMTQ4MDMtNjUiLCJpZCI6ImxheWVyLTIiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9hcHBzL3N0b3JhZ2UvbG90dXN3b3JrL2xvdHVzd29ya2VyL2NhY2hlL3MtdDAxNDgwMy02NSIsImlkIjoibGF5ZXItMyIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL2FwcHMvc3RvcmFnZS9sb3R1c3dvcmsvbG90dXN3b3JrZXIvY2FjaGUvcy10MDE0ODAzLTY1IiwiaWQiOiJsYXllci00Iiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvYXBwcy9zdG9yYWdlL2xvdHVzd29yay9sb3R1c3dvcmtlci9jYWNoZS9zLXQwMTQ4MDMtNjUiLCJpZCI6ImxheWVyLTUiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9hcHBzL3N0b3JhZ2UvbG90dXN3b3JrL2xvdHVzd29ya2VyL2NhY2hlL3MtdDAxNDgwMy02NSIsImlkIjoibGF5ZXItNiIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL2FwcHMvc3RvcmFnZS9sb3R1c3dvcmsvbG90dXN3b3JrZXIvY2FjaGUvcy10MDE0ODAzLTY1IiwiaWQiOiJsYXllci03Iiwic2l6ZSI6MTA3Mzc0MTgyNCwicm93c190b19kaXNjYXJkIjo3fSx7InBhdGgiOiIvYXBwcy9zdG9yYWdlL2xvdHVzd29yay9sb3R1c3dvcmtlci9jYWNoZS9zLXQwMTQ4MDMtNjUiLCJpZCI6ImxheWVyLTgiLCJzaXplIjoxMDczNzQxODI0LCJyb3dzX3RvX2Rpc2NhcmQiOjd9LHsicGF0aCI6Ii9hcHBzL3N0b3JhZ2UvbG90dXN3b3JrL2xvdHVzd29ya2VyL2NhY2hlL3MtdDAxNDgwMy02NSIsImlkIjoibGF5ZXItOSIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL2FwcHMvc3RvcmFnZS9sb3R1c3dvcmsvbG90dXN3b3JrZXIvY2FjaGUvcy10MDE0ODAzLTY1IiwiaWQiOiJsYXllci0xMCIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N30seyJwYXRoIjoiL2FwcHMvc3RvcmFnZS9sb3R1c3dvcmsvbG90dXN3b3JrZXIvY2FjaGUvcy10MDE0ODAzLTY1IiwiaWQiOiJsYXllci0xMSIsInNpemUiOjEwNzM3NDE4MjQsInJvd3NfdG9fZGlzY2FyZCI6N31dLCJfaCI6bnVsbH19LCJjb25maWciOnsicGF0aCI6Ii9hcHBzL3N0b3JhZ2UvbG90dXN3b3JrL2xvdHVzd29ya2VyL2NhY2hlL3MtdDAxNDgwMy02NSIsImlkIjoidHJlZS1kIiwic2l6ZSI6MjE0NzQ4MzY0Nywicm93c190b19kaXNjYXJkIjo3fSwiY29tbV9kIjpbNywxMjYsOTUsMjIyLDUzLDE5NywxMCwxNDcsMywxNjUsODAsOSwyMjcsNzMsMTM4LDc4LDE5MCwyMjMsMjQzLDE1Niw2NiwxODMsMTYsMTgzLDQ4LDIxNiwyMzYsMTIyLDE5OSwxNzUsMTY2LDYyXX0=","TicketValue":"Po0Uc9aw55gJEg5Zusi/VZmO6yJPrZdPAtEDGgtAODI=","TicketEpoch":180985}}
7531.	2020-10-28 10:28:12 +0800 CST:	[event;sealing.SectorPreCommit2]	{"User":{"Sealed":{"/":"bagboea4b5abcakcv3wuyhegkproomyfyhfwzf4muuu67opuxwtk5dovn43j3lakj"},"Unsealed":{"/":"baga6ea4seaqao7s73y24kcutaosvacpdjgfe5pw76ooefnyqw4ynr3d2y6x2mpq"}}}
7532.	2020-10-28 10:28:12 +0800 CST:	[event;sealing.SectorChainPreCommitFailed]	{"User":{}}
	pushing message to mpool: GasEstimateMessageGas error: estimating gas used: message execution failed: exit 16, reason: seal challenge epoch 180985 too old, must be after 181071 (RetCode=16)
7533.	2020-10-28 10:29:12 +0800 CST:	[event;sealing.SectorRetryPreCommit]	{"User":{}}
7534.	2020-10-28 10:29:12 +0800 CST:	[event;sealing.SectorChainPreCommitFailed]	{"User":{}}
	pushing message to mpool: GasEstimateMessageGas error: estimating gas used: message execution failed: exit 16, reason: seal challenge epoch 180985 too old, must be after 181073 (RetCode=16)
7535.	2020-10-28 10:30:12 +0800 CST:	[event;sealing.SectorRetryPreCommit]	{"User":{}}
7536.	2020-10-28 10:30:12 +0800 CST:	[event;sealing.SectorChainPreCommitFailed]	{"User":{}}
	pushing message to mpool: GasEstimateMessageGas error: estimating gas used: message execution failed: exit 16, reason: seal challenge epoch 180985 too old, must be after 181077 (RetCode=16)
7537.	2020-10-28 10:31:12 +0800 CST:	[event;sealing.SectorRetryPreCommit]	{"User":{}}
7538.	2020-10-28 10:31:12 +0800 CST:	[event;sealing.SectorChainPreCommitFailed]	{"User":{}}
	pushing message to mpool: GasEstimateMessageGas error: estimating gas used: message execution failed: exit 16, reason: seal challenge epoch 180985 too old, must be after 181080 (RetCode=16)

**Lotus miner logs**

** Code modifications **

If you have modified parts of lotus, please describe which areas were modified,
and the scope of those modifications

**Version**

lotus version 1.1.2+git.d4cdc6d3

**Comment by jennijuju** [comment]
check if your  mempool, is it stuck and not able to  send out proof messages out in time?
